.github/workflows/build-packaging.yml

```diff
@@ -276,7 +276,7 @@ jobs:
         with:
           manifest-file-path: ${{ github.workspace }}/manifest.json
           default-target-path: .pax/binaryDependencies/
-          expected-count: 28
+          expected-count: 30
```

Comment:
Missing update in https://github.com/zowe/zowe-install-packaging/tree/v3.x/staging/bin#utilities